docs/content/en/install/gke/_index.md

```diff
@@ -339,22 +469,24 @@ WARNING: version difference between client (1.21) and server (1.18) exceeds the
 
 ## Install K8ssandra
 
-With all of the infrastructure provisioned we can now focus on installing K8ssandra. This will require configuring a service account for the backup and restore service, creating a set of Helm variable overrides, and setting up GKE specific ingress configurations.
+With all of the infrastructure provisioned we can now focus on installing K8ssandra. This will require configuring a service account for the backup and restore service (Medusa), creating a set of Helm variable overrides, and setting up GKE specific ingress configurations.
 
 ### Create Backup / Restore Service Account Secrets
 
 In order to allow for backup and restore operations, we must create a service account for the Medusa operator which handles coordinating the movement of data to and from Google Cloud Storage (GCS) buckets. As part of the provisioning sections a service account was generated for this purposes. Here we will retrieve the authentication JSON file for this account and push it into Kubernetes as a secret.
 
 Looking at the output of `terraform plan` and `terraform apply` we can see the name of the service account which has been provisioned. Here we use `terraform output` to retrieve keys for use by Medusa. In our reference implementation this value is `prod-k8ssandra-sa@k8ssandra-testing.iam.gserviceaccount.com`.
 
+Because Medusa will expect a JSON file to pass in the secret, we'll use `-raw` to avoid a Terraform output issue: 
+
 ```console
-terraform output -json service_account_key > medusa_gcp_key.json
+terraform output -raw service_account_key > medusa_gcp_key
```

Comment:
I don't understand why the text says Medusa expects JSON and then we ask terraform for "raw" output?

Reply:
maybe it needs to say something like, we'll put it in a raw file first and then put it in JSON